emoji/src/main/java/com/vanniktech/emoji/EmojiReplacer.java

```diff
@@ -0,0 +1,8 @@
+package com.vanniktech.emoji;
+
+import android.content.Context;
+import android.text.Spannable;
+
+public interface EmojiReplacer {
```

Comment:
some java doc what it can be used for and also that it'll be there since version 0.6.0